Control coverage/Feature/KubernetesService.md

```diff
@@ -388,3 +388,36 @@ Enabling HTTP application routing creates publicly accessible DNS names for appl
 
 ___ 
 
+
+## Azure_KubernetesService_AuthN_Disable_Local_Accounts
+
+### Display Name
+Local accounts should be disabled in Kubernetes Service.
+
+### Rationale
+Disable local accounts with AAD authentication enabled on the kubernete cluster to avoid non-auditable access to the cluster access and improve cluster security.
+
+### Control Spec
+
+> **Passed:**
+> Local accounts is disabled on the Kubernetes cluster.
+>
+> **Failed:**
+> Local accounts is enabled on the Kubernetes cluster.
+>
+### Recommendation
+
+- **Azure Portal**
+
+	Go to Azure Portal --> Kubernetes Services --> Select Kubernetes Cluster --> Settings --> Cluster configuration --> Authentication and Authorization options -->  Authentication and Authorization option --> Select 'Azure AD Authentication with Kubernetes RBAC' --> Cluster admin ClusterRoleBinding option --> Click 'choose AAD Group' --> select group and click Select --> Uncheck Kubernetes local accounts option --> Select Apply.
```

Comment:
Authentication and Authorization options - Is this repeated or is this the actual order. can't verify as there is no AKS in -02 rn

Reply:
It is the actual order.